src/db/mod.rs

```diff
@@ -492,4 +493,15 @@ impl Database {
             .execute_watch_with_session(pipeline, options, target, None, session)
             .await
     }
+
+    /// Creates a new [`GridFsBucket`] in the database with the given options.
+    pub fn new_gridfs_bucket(
+        &self,
+        options: impl Into<Option<GridFsBucketOptions>>,
+    ) -> GridFsBucket {
+        GridFsBucket {
+            db: self.clone(),
+            options: options.into(),
```

Comment:
I think it would be a good idea to include to store the fields specified in the `GridFsBucketOptions` struct directly within `GridFsBucket`, especially since `chunk_size_bytes` and `bucket_name` are used frequently in the upload and download methods and it makes the code harder to read to figure out those values from all the layers of `Option` around those fields.